res/controllers/midi-controller-api.d.ts

```diff
@@ -32,5 +32,5 @@ declare namespace midi {
 
     type InputCallback = (channel: string, control: string, value: number, status: number) => void
 
-    function makeInputHandler(status: number, midino: number, callback: InputCallback): MidiInputHandlerController
+    function makeInputHandler(status: number, midino: number, callback: InputCallback): MidiInputHandlerController | undefined
```

Comment:
when would it return `undefined`? It either succeeds or throws, right?

Reply:
Right. It's a bit counter-intuitif since the C++ implementation still returns an undefined value.

Reply:
yeah... but I think that is simply swallowed essentially. So I think we can omit that detail from the API description. wdyt?